src/coreclr/jit/gentree.cpp

```diff
@@ -24618,6 +24632,20 @@ ClassLayout* GenTreeLclVarCommon::GetLayout(Compiler* compiler) const
     return AsLclFld()->GetLayout();
 }
 
+//------------------------------------------------------------------------
+// GenTreeLclVar::IsNeverNegative: Gets true if the lcl var is never negative; otherwise false.
+//
+// Arguments:
+//    comp - the compiler instance
+//
+// Return Value:
+//    true if the lcl var is never negative; otherwise false.
+//
+bool GenTreeLclVar::IsNeverNegative(Compiler* comp) const
+{
+    return comp->lvaGetDesc(GetLclNum())->IsNeverNegative();
```

Comment:
It also needs either `assert(OperIs(GT_LCL_VAR))` or `return OperIs(GT_LCL_VAR) ? { ... } : false`, since the node is used for stores as well as `GT_LCL_VAR_ADDR`.

Reply:
Fixed.